gtsfm/utils/tracks.py

```diff
@@ -0,0 +1,72 @@
+"""Utilities for 2D and 3D tracks.
+
+Authors: Ayush Baid
+"""
+from typing import Dict, List
+
+from gtsam import Cal3Bundler, SfmTrack
+
+from gtsfm.common.sfm_track import SfmMeasurement, SfmTrack2d
+from gtsfm.data_association.point3d_initializer import (
+    Point3dInitializer,
+    TriangulationParam,
+    TriangulationExitCode,
+)
+
+
+def classify_tracks2d_with_gt_cameras(
```

Comment:
looks great, Ayush. Can we add a unit test for this one?

Reply:
I thought about mocking the `Point3DInitializer` and then checking the function calls, but I am afraid it will take some time. I do not want to use an example and run actual triangulation because that can create flaky tests. What do you think? @johnwlambert